w3af/core/data/context/context/main.py

```diff
@@ -78,16 +82,21 @@ def get_context_iter(data, payload):
 
 class ContextDetectorHTMLParser(HTMLParser):
 
-    def __init__(self, payload):
+    RAW_TEXT_TAG = {
+        'title',
+        'textarea',
+        'plaintext',
+        'xmp',
+        'listing'
+    }
```

Comment:
Because It handles raw content (`RCDATA`, `RAWTEXT`, etc) and not parsed nested HTML-tags, expect end tag.
In other words:
```html
This is NOT a XSS vulnerability
<title><svg onload=prompt`XSS`></title>

This is it
<title></title><svg onload=prompt`XSS`></title>
```
So for these tags I created new context `HtmlRawText` which can be breaks by `</`